<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>695</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <italic>false</italic>
    <underline>false</underline>
    <strikeout>false</strikeout>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: #00111d;
	color: #000000;
	border-color: #000000;

}


QPushButton {
    height: 50px;
    width: 120px;
    background-color: #333;
    border: 1px solid #404c5d;
    color: rgb(232, 232, 232);
    font-family: Consolas, &quot;Courier New&quot;, monospace;
    font-size: 14px;
    border-radius: 5px;

    /* Gradient background */
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #2e2d2d,
        stop:1 #212121
    );

    padding: 5px;
}

/* Hover */
QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #353535,
        stop:1 #262626
    );
    color: #d6d6d6;
}

/* Pressed */
QPushButton:pressed {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #262626,
        stop:1 #1a1a1a
    );
    color: #e0e0e0;

    /* Slight inset effect */
    padding-top: 7px;
    padding-bottom: 3px;
}

QTableWidget {
        background-color: #001622;
        color: white;
        font-size: 14px;
        border: 1px solid #4c4c4c;
    }

QHeaderView::section {
        background-color: #0b2e4a;
        color: white;
        padding: 5px;
        border: 1px solid #32414B;
        font-weight: bold;
    }

QTableWidget::item:selected {
        background-color: #1e90ff;
        color: black;
    }

QLineEdit
{
	background-color: #082336;
	color: #3e93ff;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

}


QLineEdit::hover
{
	background-color: #082336;
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}


QLineEdit::focus
{
	
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}

QMessageBox {
    color: white;
}
QLabel{
    font-family: Consolas, &quot;Courier New&quot;, monospace;
	color: rgb(232, 232, 232);
	font: 10pt;
}
QPushButton {
    color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>500</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas,Courier New,monospace</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    height: 50px;
    width: 120px;
    background-color: #333;
    border: 1px solid #404c5d;
    color: rgb(232, 232, 232);
    font-family: Consolas, &quot;Courier New&quot;, monospace;
    font-size: 18px;
    border-radius: 10px;

    /* Gradient background */
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #2e2d2d,
        stop:1 #212121
    );

    padding: 5px;
}

/* Hover */
QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #353535,
        stop:1 #262626
    );
    color: #d6d6d6;
}

/* Pressed */
QPushButton:pressed {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #262626,
        stop:1 #1a1a1a
    );
    color: #e0e0e0;

    /* Slight inset effect */
    padding-top: 7px;
    padding-bottom: 3px;
}
</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas,Courier New,monospace</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    font-family: Consolas, &quot;Courier New&quot;, monospace;
	color: rgb(232, 232, 232);
	font: 20pt;
}</string>
    </property>
    <property name="text">
     <string>University ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas,Courier New,monospace</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    font-family: Consolas, &quot;Courier New&quot;, monospace;
	color: rgb(232, 232, 232);
	font: 20pt;
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDlineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasslineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icons/login-user-name-1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_show">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox
{
	color: #b1b1b1;
    background-color: #00111d;
    width: 12px;
    height: 12px;

}

</string>
    </property>
    <property name="text">
     <string>Show Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SignUpButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>570</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas,Courier New monospace</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    height: 50px;
    width: 120px;
	background-color: rgb(0, 17, 29);
    color: rgb(232, 232, 232);
    font-family: Consolas, &quot;Courier New&quot;monospace;
    font-size: 12px;
    border-radius: 10px;
	border-color: rgb(0, 17, 29);

    /* Gradient background */
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #00111d,
        stop:1 #00111d
    );

    padding: 5px;
}

/* Hover */
QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #00111d,
        stop:1 #00111d
    );
    color: #d6d6d6;
}

/* Pressed */
QPushButton:pressed {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #00111d,
        stop:1 #00111d
    );
    color: #e0e0e0;

    /* Slight inset effect */
    padding-top: 7px;
    padding-bottom: 3px;
}
	</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ForgetButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>570</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas,Courier New monospace</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>Click here to reset  your password</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    height: 50px;
    width: 120px;
	background-color: rgb(0, 17, 29);
    color: rgb(232, 232, 232);
    font-family: Consolas, &quot;Courier New&quot;monospace;
    font-size: 12px;
    border-radius: 10px;
	border-color: rgb(0, 17, 29);

    /* Gradient background */
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #00111d,
        stop:1 #00111d
    );

    padding: 5px;
}

/* Hover */
QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #00111d,
        stop:1 #00111d
    );
    color: #d6d6d6;
}

/* Pressed */
QPushButton:pressed {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #00111d,
        stop:1 #00111d
    );
    color: #e0e0e0;

    /* Slight inset effect */
    padding-top: 7px;
    padding-bottom: 3px;
}
	</string>
    </property>
    <property name="text">
     <string>Forget Password?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>IDlineEdit</tabstop>
  <tabstop>PasslineEdit</tabstop>
  <tabstop>checkBox_show</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>SignUpButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
